<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777214</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lottozahlen</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../jackpot_header_swisslotto_logo.png</normaloff>
    <normalon>../../../../jackpot_header_swisslotto_logo.png</normalon>
    <disabledoff>../../../../jackpot_header_swisslotto_logo.png</disabledoff>
    <disabledon>../../../../jackpot_header_swisslotto_logo.png</disabledon>
    <activeoff>../../../../jackpot_header_swisslotto_logo.png</activeoff>
    <activeon>../../../../jackpot_header_swisslotto_logo.png</activeon>
    <selectedoff>../../../../jackpot_header_swisslotto_logo.png</selectedoff>
    <selectedon>../../../../jackpot_header_swisslotto_logo.png</selectedon>../../../../jackpot_header_swisslotto_logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(235, 172, 255, 197);
background-color: rgb(255, 251, 248);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="1" column="0">
     <spacer name="horizontalSpacer_11">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="1">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>28</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QPushButton" name="CHButton">
            <property name="maximumSize">
             <size>
              <width>201</width>
              <height>201</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton
{
background-color: transparent;
border-image: url(:);
background: none;
border: none;
background-repeat: none;
}
#pushButton:pressed
{
	border-image: url(:)
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>CH.png</normaloff>CH.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="EUButton">
            <property name="maximumSize">
             <size>
              <width>201</width>
              <height>201</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton
{
background-color: transparent;
border-image: url(:);
background: none;
border: none;
background-repeat: none;
}
#pushButton:pressed
{
	border-image: url(:)
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>EU.png</normaloff>EU.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>38</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="5" rowspan="2">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>28</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1" rowspan="2">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>28</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0" rowspan="2" colspan="2">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>38</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="4" rowspan="2" colspan="2">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>58</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0" colspan="6">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>78</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="genButton">
            <property name="font">
             <font>
              <family>Imprint MT Shadow</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 170, 255)</string>
            </property>
            <property name="text">
             <string>Lottozahlen</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>88</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="0" rowspan="2" colspan="6">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>198</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>178</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">#pushButton
{
background-color: transparent;
border-image: url(:);
background: none;
border: none;
background-repeat: none;
}
#pushButton:pressed
{
	border-image: url(:)
}</string>
            </property>
            <property name="text">
             <string>Lotto wählen ^</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">#pushButton
{
background-color: transparent;
border-image: url(:);
background: none;
border: none;
background-repeat: none;
}
#pushButton:pressed
{
	border-image: url(:)
}
</string>
            </property>
            <property name="text">
             <string>Lottozahlen ziehen v</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>198</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>178</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="3" column="2" rowspan="4" colspan="2">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Imprint MT Shadow</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton
{
background-color: transparent;
border-image: url(:);
background: none;
border: none;
background-repeat: none;
}
#pushButton:pressed
{
	border-image: url(:)
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="Zahl_1">
            <property name="font">
             <font>
              <family>Imprint MT Shadow</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton
{
background-color: transparent;
border-image: url(:);
background: none;
border: none;
background-repeat: none;
}
#pushButton:pressed
{
	border-image: url(:)
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="Zahl_2">
            <property name="font">
             <font>
              <family>Imprint MT Shadow</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton
{
background-color: transparent;
border-image: url(:);
background: none;
border: none;
background-repeat: none;
}
#pushButton:pressed
{
	border-image: url(:)
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="Zahl_3">
            <property name="font">
             <font>
              <family>Imprint MT Shadow</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton
{
background-color: transparent;
border-image: url(:);
background: none;
border: none;
background-repeat: none;
}
#pushButton:pressed
{
	border-image: url(:)
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="Zahl_4">
            <property name="font">
             <font>
              <family>Imprint MT Shadow</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton
{
background-color: transparent;
border-image: url(:);
background: none;
border: none;
background-repeat: none;
}
#pushButton:pressed
{
	border-image: url(:)
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="Zahl_5">
            <property name="font">
             <font>
              <family>Imprint MT Shadow</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton
{
background-color: transparent;
border-image: url(:);
background: none;
border: none;
background-repeat: none;
}
#pushButton:pressed
{
	border-image: url(:)
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QLabel" name="Zahl_6">
            <property name="font">
             <font>
              <family>Imprint MT Shadow</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton
{
background-color: transparent;
border-image: url(:);
background: none;
border: none;
background-repeat: none;
}
#pushButton:pressed
{
	border-image: url(:)
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_12">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
